sig-architecture/api-review-process.md

```diff
@@ -85,94 +61,83 @@ Because expert reviewer bandwidth is extremely limited, the process provides a c
+      * The approved issue is archived in the review project board, and the `api-review` label is removed.
+    * Changes requested:
+      * Comments or questions are left on the PR or issue, and the reviewer notifies the submitter
+      * The reviewer moves the issue to "Changes Requested" in the review project board
+      * Once the requested changes are made, or questions are answered, the submitter notifies the reviewer, who moves the issue back to "In Progress"
+      * To the degree possible, complete sets of comments/changes should be requested and made, to avoid excessive back-and-forth cycles.
+    * Rejected:
+      * If completely rejected, e.g. "please do this work outside the Kubernetes org" - an explanation of why the change was rejected - appeals can be requested from the api-approvers mailing list ([kubernetes-api-reviewers@googlegroups.com](mailto:kubernetes-api-reviewers@googlegroups.com)) where the moderator will coordinate a follow-up review.  If that request results in another rejection, there is no further appeal.
+      * The rejected issue is archived in the review project board, and the `api-review` label is removed.
+
+# Open improvements
+
+To enable this process, the following needs to be done:
+
+* Add `/api-review` and `/api-review cancel` bot commands to allow any org member to request or cancel a request for an API review
+
+To improve visibility and understanding of this process, the following would be helpful:
+
+* Add bot logic to automatically leave a comment in any PR flagged with `kind/api-change` outlining the prereqs and steps for requesting a review
 
 ## Review lifecycle timing
 
 Ideally, reviews will happen as quickly as possible, but it is highly dependent on reviewer availability and bandwidth. In general, the following timeframe can be expected:
 
-- time t: create PR in api-reviews
+- time t: request review
 - time t+1 week: prioritized and queued
 - time t+3 weeks: first review complete
 - time t+4 weeks: subsequent review complete
 - time t+6 weeks: approved or rejected
 
+Timing of API review requests matters.  The larger the change the more time that must be afforded.  New API resources (aka Kinds) may require significantly more thought than single field additions.  API reviews that are requested too late in a release cycle may not complete in time to make the release.  Plan ahead. Also, if you are changing an approved API, you must consult with the [kubernetes-api-reviewers@googlegroups.com](mailto:kubernetes-api-reviewers@googlegroups.com) list to ensure it is still consistent with the approvals already granted. From a process perspective, you would request a new review in that case.
+
+## The Moderator Role
```

Comment:
We may want to drop this because so far it has been unstaffed. Maybe add a to-do here once more people are actively working on/in this process, especially at the SIG level.

Reply:
this was just a move of the moderators section... let's leave it as-is for this PR, rather than removing it, especially if sig-pm is potentially interested in helping here